app/src/main/java/com/example/cocktailmaster/ui/home/HomeFragment.kt

```diff
@@ -88,10 +94,11 @@ class HomeFragment :
             R.id.imageSearch -> binding.searchDrinks.show()
 
             R.id.textSeeAllCategory -> fragmentManager?.let {
-                replaceFragment(
-                    it,
-                    CategoryFragment()
-                )
+                replaceFragment(it, CategoryFragment())
+            }
+
+            R.id.textSeeAllIngredient -> fragmentManager?.let {
+                replaceFragment(it, IngredientFragment())
             }
```

Comment:
ngoặc kép

Reply:
ngoac kep la sao a , e chua hiu lam

Reply:
biểu thức trong lệnh when mà k thể viết ở 1 dòng thì đều cho vào ngoặc kép nhé
```suggestion
            R.id.textSeeAllIngredient -> { fragmentManager?.let { .....
               
            }
```

Reply:
em da sua lai r nha